### PR TITLE
Stripe

### PR DESCRIPTION
**Pull request by Monkeyank**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Add Stripe Completion Spec
**What is the current behavior? (You can also link to an open issue here)**
#307 
**What is the new behavior (if this is a feature change)?**

**Additional info:**

### CODE REVIEW

File: dev/stripe.ts [hunk 1]
```diff
@@ -0,0 +1,759 @@
...
+      name: "setting",
+      suggestions: ["on", "off", "auto"],
+    },
+  },
+  {
+    name: "--config",
+    description: "Sets your config file",
+    args: {
+      name: "config filepath",
+    },
```
Comment (QuiiBz) [inline dev/stripe.ts:22]: I think this can have a template of `filepaths`

File: dev/stripe.ts [hunk 2]
```diff
@@ -0,0 +1,759 @@
...
+const sharedOptions: Fig.Option[] = [
+  {
+    name: ["-s", "--show-headers"],
+    description: "Shows response HTTP headers",
+  },
+  {
+    name: ["-c", "--confirm"],
+    description:
+      "Skips the warning prompt and automatically confirms the command being entered",
+  },
```
Comment (QuiiBz) [inline dev/stripe.ts:59]: We can add `isDangerous` here

File: dev/stripe.ts [hunk 3]
```diff
@@ -0,0 +1,759 @@
...
+            description: "Key1:Value1, Key2:Value2",
+          },
+        },
+        {
+          name: ["-e", "--events"],
+          description: "A comma-seperated list of which events to listen for",
+          args: {
+            variadic: true,
+            name: "events types",
+          },
```
Comment (QuiiBz) [inline dev/stripe.ts:449]: Could this arg provide suggestions for the list of events? Looks like they are defined here: https://github.com/withfig/autocomplete/pull/308/files#diff-e38baeee0f96b87459e1901e70e24b9f815f6a3a4ddd9b7e27f811f7dadf4408R508

File: dev/stripe.ts [hunk 4]
```diff
@@ -0,0 +1,759 @@
...
+            "Listen for all webhook events based on existing webhook endpoints",
+        },
+        {
+          name: ["-j", "--print-json"],
+          description: "Print JSON objects to stdout",
+        },
+        {
+          name: "--skip-verify",
+          description:
+            "Skip certificate verification when forwarding to HTTPS endpoints",
```
Comment (QuiiBz) [inline dev/stripe.ts:496]: We can also add `isDangerous` here

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Awesome CLI! I left some comments

**Comment by QuiiBz** [comment]
Are those suggestions the same?

https://github.com/withfig/autocomplete/pull/308/files#diff-e38baeee0f96b87459e1901e70e24b9f815f6a3a4ddd9b7e27f811f7dadf4408R451-R1235

https://github.com/withfig/autocomplete/pull/308/files#diff-e38baeee0f96b87459e1901e70e24b9f815f6a3a4ddd9b7e27f811f7dadf4408R1296-R1342

**Comment by Monkeyank** [comment]
> Are those suggestions the same?
> 
> https://github.com/withfig/autocomplete/pull/308/files#diff-e38baeee0f96b87459e1901e70e24b9f815f6a3a4ddd9b7e27f811f7dadf4408R451-R1235
> 
> https://github.com/withfig/autocomplete/pull/308/files#diff-e38baeee0f96b87459e1901e70e24b9f815f6a3a4ddd9b7e27f811f7dadf4408R1296-R1342

They're not the same, because one of them is just all/general events, while the other is specific to webhook events.

**Comment by QuiiBz** [comment]
> They're not the same, because one of them is providing suggestions for all/general events, while the other is providing suggestions specific to webhook events.

Alright, then it would be better not to duplicate the webhooks events and put them in a top-level const Fig.Suggestion[]. Then, you can use this const for the webhooks events suggestions and spread it for the general events suggestions.

**Review by QuiiBz — APPROVED** [review]
LGTM!

**Pull request merged**